src/App.asm

```diff
@@ -1,6 +1,75 @@
 .data
-
+	str1: .asciiz "Enter total number of entries: "
+	str2: .asciiz "Enter an integer: "
+	totalPositive: .asciiz "Total positive "
+	totalNegative: .asciiz "Total negative "
+	
 
 .globl main
 
-.text
\ No newline at end of file
+.text
+
+main:
+
+    #print 
+    la $a0, str1
+    li $v0, 4
+    syscall
+
+    #scan the counter
+    li $v0, 5
+    syscall
+    move $t0, $v0 
+
+    # Initialize variables instead of using .word
+    li $t1, 0 # i = 0
+    li $t2, 0 # positive = 0
+    li $t3, 0 # negative = 0
+    li $t4, 0 # x = 0
+
+    #while loop 
+    loop:
+        beq $t1, $t0, endLoop 
+
+        # Print str2
+        la $a0, str2
+        li $v0, 4
+        syscall
+
+        # Read integer input
+        li $v0, 5
+        syscall
+        move $t4, $v0 # Store the input in $t4
+
+        # Check if x > 0
+        bgtz $t4, positive #branch great than zero is used here
```

Comment:
you check only the positive but the negative has not handled, and you include zero with negative values. So you program does not provide correct result (logical error)